testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -750,7 +750,16 @@ def test_nonexistant_offsets_file(self, traj):
             saved_offsets = XDR.read_numpy_offsets(outfile_offsets)
             assert_equal(saved_offsets, False)
 
-    def test_reload_offsets_if_offsets_readin_fails(self, trajectory):
+    def test_nonexistant_offsets_file(self, traj):
```

Comment:
spelling: nonexistent ([existant](https://en.wiktionary.org/wiki/existant) is a noun)

Reply:
fixed in [94611d7](https://github.com/MDAnalysis/mdanalysis/pull/3375/commits/94611d79eab7d7e289410ce2649e184581a8f471)